### DISCUSSION TITLE
ImportError: DLL load failed while importing libpaddle: The specified module could not be found.

### DISCUSSION THREAD

**Discussion by GloriaNeo** [PaddlePaddle/PaddleOCR#11399] [Q&A]
Hi, I have managed to pip install paddleocr and paddlepaddle successfully. However, when I try to import paddleocr, I will receiving this error "ImportError: DLL load failed while importing libpaddle: The specified module could not be found." 

Have searched through stackoverflow and found posts with similar issues (https://stackoverflow.com/questions/76388016/error-can-not-import-paddle-core-while-this-file-exists) have tried the solutions there but none of them seem to work for me.

Please help! Thank you!

**Comment by deeptvyas25** [comment]
Try installing VC++ redistributable 2015 - 2022, it worked for me.
